spec/order_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'order'
+
+describe Order do
+  let(:menu) {
+    double(
+      :menu,
+      name: "Dominos",
+      dishes:
```

Comment:
I would probably separate this out, so you have:
```
let(:dishes) do
  {"cheese & tomato" => 6}
end
let(:menu){ double :menu, name: "dominos", dishes: dishes }
```
Also, keep your fixtures simple - you don't need 5 dishes, just one or 2 will be easier to test and verify your tests are correct.